File: lib/AdditionalCellState/AdditionalCellState.m
//
//  AdditionalCellState.m
//  TableViewAgent
//
//  Created by P.I.akura on 2013/06/18.
//  Copyright (c) 2013年 P.I.akura. All rights reserved.
//

#import "AdditionalCellState.h"

@implementation AdditionalCellState

- (BOOL)isShowAddCell:(BOOL) editing {
    return NO;
}

- (ChangeInState)changeInState:(BOOL) editing {
    return ChangeInStateNone;
}

@end

File: lib/AdditionalCellState/AdditionalCellState.swift
//
//  AdditionalCellState.swift
//  TableViewAgent
//
//  Created by akuraru on 2014/07/29.
//  Copyright (c) 2014年 P.I.akura. All rights reserved.
//

import Foundation

enum ChangeInState {
    case None, Hide, Show
}

class AdditionalCellState :NSObject {
    func isShowAddCell(editing :Bool) -> Bool {
        return false
    }
    
    func changeInState(editing :Bool) -> Int {
        return 0;
    }

}